src/main/java/Deadline.java

```diff
@@ -0,0 +1,28 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    private String byWhen;
+    private LocalDate deadline;
+
+    public Deadline(String item, boolean marked, String byWhen) {
+        super(item, marked);
+        this.byWhen = byWhen;
+        this.deadline = LocalDate.parse(byWhen);
+    }
+
+    public String getbyWhen() {
+        return this.byWhen;
+    }
+
+    @Override
+    public String stringify() {
```

Comment:
What does `stringify `mean? I know there is a stringify fun in JavaScript for JSON things, does it also defined in java?

Reply:
I think the method they were looking for might be `toString()`? Based on how this method is used I think that would be a more appropriate name. Also, another benefit of overriding `toString()` instead of writing an entirely new method is because then Java can automatically call it whenever the object needs to be converted to a String (i.e. string concatenation or passing objects as arguments to `System.out.println()`).
```suggestion
    public String toString() {
```